plugins/catalog-backend-module-msgraph/api-report.md

```diff
@@ -233,6 +238,11 @@ export type OrganizationTransformer = (
   organization: MicrosoftGraph.Organization,
 ) => Promise<GroupEntity | undefined>;
 
+// @public
+export type ProviderTransformer = (
```

Comment:
I'm thinking `ProviderConfigTransformer` might be more explicit?

Reply:
I mostly agree with you on this one - the reason why I went when `ProviderTransformer` is because "raw" access is currently given to the `provider` itself. Mostly this was done because the concept of "config" is gone once `fromConfig()` completes, and everything lives directly within the `provider` object

Reply:
nevermind - I fully agree with you - even the `provider` extends the `Config` object - I will just update with documentation specifying any fields which cannot be modified